### PR TITLE
Yet another doxygen fix

### PR DESCRIPTION
**Pull request by OlegHahm**
However, `doc.txt` seems still not be properly evaluated. The two macros do not show up in the documentation.

Also, the sorting is somewhat weird. Will open another issue for that.

@Lotterleben, please check if you agree on the AODVv2 specific change.

@authmillenon, please add a commit to  #2614 to revert 4ee380d

### CODE REVIEW

**Comment by Lotterleben** [comment]
@OlegHahm Yes, that was a copy&paste error on my part, thank you for fixing it.

File: core/include/priority_queue.h [hunk 1]
```diff
@@ -110,7 +110,18 @@ void priority_queue_add(priority_queue_t *root, priority_queue_node_t *new_obj);
 void priority_queue_remove(priority_queue_t *root, priority_queue_node_t *node);
 
 #if ENABLE_DEBUG
+/**
+ * @brief print the data and priority of every noe in the given priority queue
```
Comment (Lotterleben) [inline core/include/priority_queue.h]: noe -> node

**Comment by OlegHahm** [comment]
fixed typo

File: core/include/debug.h [hunk 2]
```diff
@@ -14,7 +14,7 @@
  * @brief       Debug-header
  *
  * @details     If ENABLE_DEBUG is set, before this header is included,
- *              ::DEBUG will print out to stdout, otherwise do nothing
+ *              ::DEBUG_FUNC will print out to stdout, otherwise do nothing
```
Comment (Lotterleben) [inline core/include/debug.h]: I'm afraid I don't get the motivation behind this change... If I understand it correctly, DEBUG_FUNC isn't really the core function of debug.h; or did I miss something?

Comment (OlegHahm) [inline core/include/debug.h]: Yes, you're right. This change is utter nonsense. The correct fix would be to add documentation for `DEBUG`.

Comment (Lotterleben) [inline core/include/debug.h]: Yes, that would be nice. :)

Comment (authmillenon) [inline core/include/debug.h]: Wait I did this somewhere

Comment (authmillenon) [inline core/include/debug.h]: Nevermind, was `ENABLE_DEBUG` in #2429

**Comment by authmillenon** [comment]
> @authmillenon, please add a commit to #2614 to revert 4ee380d

Is 4ee380d really necessary appart from doxygen complaining until #2614 got merged?

File: core/include/debug.h (hunk 2 above)
Comment (OlegHahm) [inline core/include/debug.h]: It turned out that this needed even more cleanup as expected.

**Comment by OlegHahm** [comment]
> >  @authmillenon, please add a commit to #2614 to revert 4ee380d

> Is 4ee380d really necessary appart from doxygen complaining until #2614 got merged?

I would say so, because it screws up the module overview: http://doc.riot-os.org/modules.html

**Comment by authmillenon** [comment]
Okay, then I do this as soon as this PR got merged

File: core/include/debug.h [hunk 3]
```diff
@@ -94,7 +91,20 @@ extern "C" {
...
+ * @brief Print debug information to std-out
+ *
+ * @note Another name for ::DEBUG_PRINT
+ */
 #define DEBUG(...) DEBUG_PRINT(__VA_ARGS__)
+/**
+ * @def DEBUGF
+ *
+ * @brief Print extended debug information about the current thread and
+ *        function to sted-out
```
Comment (Lotterleben) [inline core/include/debug.h]: tiny nitpick: shouldnt it say ``stdout`` instead of ``sted-out``?

Comment (OlegHahm) [inline core/include/debug.h]: updated

**Comment by Lotterleben** [comment]
Thanks @OlegHahm, could you please squash? :)

**Comment by OlegHahm** [comment]
I can. :wink:

**Comment by Lotterleben** [comment]
Thanks! ACK as soon as Travis is happy.

**Comment by authmillenon** [comment]
This might take a while… just had to rebase most of my PRs...

**Comment by OlegHahm** [comment]
Do we really need to run all the Travis checks for these PRs?

**Comment by LudwigOrtmann** [comment]
Could you change these to `@ingroup` while you're at it?
```
git grep core_util | grep addto
core/include/atomic.h: * @addtogroup  core_util
core/include/bitarithm.h: * @addtogroup  core_util
core/include/byteorder.h: * @addtogroup     core_util
core/include/cib.h: * @addtogroup  core_util
core/include/clist.h: * @addtogroup  core_util
core/include/crash.h: * @addtogroup  core_util
core/include/debug.h: * @addtogroup  core_util
core/include/kernel_macros.h: * @addtogroup  core_util
core/include/kernel_types.h: * @addtogroup  core_util
core/include/lifo.h: * @addtogroup  core_util
core/include/priority_queue.h: * @addtogroup  core_util
```

**Comment by OlegHahm** [comment]
Why?

**Comment by authmillenon** [comment]
(Though technically correct (since `core_util` is not a non-unique group [1]) I'm of the opinion that this can be done as a follow-up, since this PR was already ACK'd)

[1] http://www.stack.nl/~dimitri/doxygen/manual/grouping.html#modules

**Comment by OlegHahm** [comment]
Can you rephrase? I'm confused by the double negative.

**Comment by authmillenon** [comment]
I was confused by the doc of doxygen again... So forget what I said: The use of `@addtogroup` is correct here! (We don't want to add a submodule to the module `core_util` (what `@ingroup` is for), but want to add the file to the scope of the group `core_util` (what `@addtogroup` is for)).
